backend/middlewares/auth.ts

```diff
@@ -10,13 +10,9 @@ import { Role } from "../types";
 const authService: IAuthService = new AuthService(new UserService());
 
 export const getAccessToken = (req: Request): string | null => {
-  const authHeaderParts = req.headers.authorization?.split(" ");
-  if (
-    authHeaderParts &&
-    authHeaderParts.length >= 2 &&
-    authHeaderParts[0].toLowerCase() === "bearer"
-  ) {
-    return authHeaderParts[1];
+  const [scheme, token] = req.headers.authorization?.split(" ") || [];
+  if (scheme.toLowerCase() === "bearer") {
```

Comment:
If `scheme` is null does this work?

Reply:
Ah good point. I don't think it would ever come up practically but I'll fix it nonetheless. Really surprised Typescript didn't find this one though...